youtube_dl/extractor/generic.py

```diff
@@ -208,12 +209,13 @@ class GenericIE(InfoExtractor):
             'playlist': [{
                 'info_dict': {
                     'ext': 'mov',
-                    'id': 'pdv_maddow_netcast_mov-12-04-2020-224335',
+                    'id': 'pdv_maddow_netcast_mov-12-11-2020-225421',
                     'title': 're:MSNBC Rachel Maddow',
                     'description': 're:.*her unique approach to storytelling.*',
                     'timestamp': int,
-                    'upload_date': compat_str,
+                    'upload_date': '20201211',
                     'duration': float,
+                    'thumbnail': None,
```

Comment:
This make zero sense. You must add a test that will test the changes made.

Reply:
@dstftw yes, you are right. Can I add another RSS feed, because this one don't get thumbnail image ?
Or I can update this one http://www.hellointernet.fm/podcast?format=rss ?

Reply:
You can use any existing RSS test with thumbnail available or add a new if there are no such tests.